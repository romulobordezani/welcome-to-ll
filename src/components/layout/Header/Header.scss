.header {
  background: #0083ca;
  display: flex;
  align-items: center;
  flex-direction: row;
  height: 90px;
  box-shadow: 0 4px 9px -1px rgba(0,0,0,0.53);
  z-index: 3;
  position: relative;
  opacity: 0; /*  << Bring it back */

  &:after{
    content: "";
    background: #02659B url("images/strip-multi-colors.png");
    background-size: contain;
    height: 4px;
    width: 100%;
    position: absolute;
    bottom: 0;
  }

  &__logo {
    flex: 0 1 auto;
    margin: 0 20px 0 0;
    img {
      max-height: 80px;
    }

  }

  &__logotype {
    align-items: center;
    color: #FFFFFF;
    font-size: 1.2em;
    -webkit-font-smoothing: antialiased;
    letter-spacing: 0.1em;

    @media( max-width: 320px ){
      font-size: 1em;
    }
  }

}
