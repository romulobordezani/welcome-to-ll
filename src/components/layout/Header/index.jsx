import React, { Component } from 'react';
import styles from './Header.scss';
import logo from './images/logo.png';

class Header extends Component {
  constructor(props) {
    super(...props);
  }

  render() {
    return (
      <header className={styles.header}>
        <div className={styles.header__logo}>
          <img src={logo} alt="Consulta de endereço" />
        </div>
        <div className={styles.header__logotype}>Consulta de endereço</div>
      </header>
    );
  }
}

export default Header;

// TODO > What about a stick header?
