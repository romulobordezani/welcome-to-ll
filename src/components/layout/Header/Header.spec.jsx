import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('[Header]', () => {
  it('Renders as the last snapshot', () => {
    const wrapper = renderer.create(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
