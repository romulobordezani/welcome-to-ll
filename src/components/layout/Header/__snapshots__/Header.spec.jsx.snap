// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Header] Renders as the last snapshot 1`] = `
<header
  className={undefined}
>
  <div
    className={undefined}
  >
    <img
      alt="Consulta de endereço"
      src="logo.png"
    />
  </div>
  <div
    className={undefined}
  >
    Consulta de endereço
  </div>
</header>
`;
